pkgs/by-name/ma/magicard-cups-driver/package.nix

```diff
@@ -0,0 +1,171 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  cmake,
+  cups,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "magicard-cups-driver";
+  version = "1.4.0";
+
+  src = fetchurl {
+    url = "https://f08ddbe93aa02eaf9a6c-f08cd513e3a8c914f4f8f62af1786149.ssl.cf3.rackcdn.com/magicard_ltd-linux_driver-${version}.tar.gz";
+    hash = "sha256-OAdpxnGMqVcf3bEtYQ9PACZFJvq8gzhGxuBBnliwoso=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ cups ];
+
+  # Replace the supplied cmake generated makefile (which is useless on a different machine)
+  # with a modified CMakeLists.txt taken from v1.3.4 of the driver
+  # (available here https://techs.magicard.com/linux/v1.3.4/magicard_ltd-linux_driver-1.3.4.tar.gz)
+  postPatch = ''
+    ln -s ${./CMakeLists.txt} CMakeLists.txt
```

Comment:
Perhaps instead of vendoring the entire CMakeLsits file and making the code opaque we could just add a patch modifying the 1.3.4 one